[Excel] (Workbook) Add onActivated conceptual docs (#2732)
OfficeDev/office-js-docs-pr: docs/excel/excel-add-ins-workbooks.md

context.application.suspendApiCalculationUntilNextSync();
```

## Detect workbook activation (preview)

> [!NOTE]
> The `Workbook.onActivated` event is currently only available in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
> 

> Your add-in can detect when a workbook is activated. After a workbook is opened, it will become "inactive" if the user switches focus to another workbook, to another application, or (in Excel on the web) to another tab of the web browser. A workbook is "activated" when the user switches focus back to this workbook. After detecting workbook activation, your add-in can trigger callback functions, such as refreshing workbook data.

Reviewer: Some phrasing suggestions. Also, I think that bold or italics are better for emphasis, rather than quotes.
```suggestion
Your add-in can detect when a workbook is activated. A workbook becomes **inactive** when the user switches focus to another workbook, to another application, or (in Excel on the web) to another tab of the web browser. A workbook is **activated** when the user returns focus to the workbook. The workbook activation can trigger callback functions your add-in, such as refreshing workbook data.
```

Author: The quotation marks are intended to indicate that these words are part of the API name. The [Style Guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28768) is kind of ambiguous about whether quotation marks are recommended in this type of scenario. I don't want to use bold for emphasis because that usually represents content in the UI (a UI button or worksheet name). I will try switching to italics.

To detect when a workbook is activated, [register an event handler](excel-add-ins-events.md#register-an-event-handler) for the [onActivated](/javascript/api/excel/excel.workbook#onActivated) event of a workbook. Event handlers for the `onActivated` event receive a [WorkbookActivatedEventArgs](/javascript/api/excel/excel.workbookactivatedeventargs) object when the event fires.

> [!IMPORTANT]
> The `onActivated` event doesn't detect when a workbook is opened. This event only detects when a user switches focus back to an already open workbook.

The following code sample shows how to register the `onActivated` event handler and set up a callback function.

```js
Excel.run(function (context) {
    // Retrieve the workbook.
    var workbook = context.workbook;

    // Register the workbook activated event handler.
    workbook.onActivated.add(workbookActivated);

    return context.sync();
});

function workbookActivated(event) {
  Excel.run(function (context) {
    // Callback function for when the workbook is activated.

>     console.log("The workbook was activated.");

Reviewer: Should this include the workbook's name?
